can-value.md

```diff
@@ -38,61 +38,37 @@
 At its simplest, [can-value.with] can be used to
 create an observable from another initial value:
 
-```js
-import value from "can-value";
-
-const observable = value.with(15);
-
-observable.value; // is 15
-```
+@sourceref ./doc/examples/observableNumber.js
+@codepen
+@highlight 3,only
 
 You can use [can-reflect/observe.onValue] to listen for
```

Comment:
I'd probably remove this and say something like:

"Notice that `canReflect.onValue()` can be used to listen for changes"